modules/gitops-uninstall-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+[id="uninstalling-keycloak_{context}"]
+= Uninstalling Keycloak
+
+You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing, the SSO field looks like the following:
+
+[source,yaml]
+----
+  apiVersion: argoproj.io/v1alpha1
+  kind: ArgoCD
+  metadata:
+    name: example-argocd
+    labels:
+      example: basic
+  spec:
+    server:
+      route:
+       enabled: true
+----
+
+[NOTE]
+====
+A Keycloak application created by this feature is currently not persistent. In case of restarts, any additional configuration created in ArgoCD Keycloak realm is deleted.
```

Comment:
Missing article:

`In case of restarts, any additional configuration created in **the** Argo CD Keycloak realm is deleted.` 

Also, I think there should be a space between Argo CD. At least, you always have that space in all other instances when you mention it.